### ISSUE TITLE
Row selection bootstrap4 style

### ISSUE THREAD

**Issue by pjvandam** [rstudio/DT#601]
I just started creating an shiny application based on the bs4Dash package to get a Bootstrap 4 version of a shiny dashboard. When selecting style = "bootstrap4", I can render tables using the bootstrap lay-out. However, when trying to select rows, the selected rows are not colored when selected. This seems to replicate an older issue with bootstrap #94 . 

Minimal example:

```R
library(shiny)
library(DT)

shinyApp(
  ui = fluidPage(DT::dataTableOutput('tbl')),
  server = function(input, output) {
    output$tbl = DT::renderDataTable(
      iris, style = "bootstrap4", options = list(lengthChange = FALSE)
    )
  }
)
```

**Comment by shrektan** [comment]
It's because DT currently doesn't use the [Select extension](
https://datatables.net/extensions/select) to do the select operations. The reason is [Select extension](
https://datatables.net/extensions/select) doesn't exist at that time.

So the style issue may be solved after DT re-implements the selecting by using the Select extension directly.

**Comment by shrektan** [comment]
Sorry, my last comment is not correct and it will be fixed by PR #765. However, you are not supposed to use bootstrap4 theme like that because shiny uses the bootstrap3 theme at present. If you are interested please read the comment in the code of the PR. 

Thanks for the reporting.
